components/cli/vendor/github.com/docker/docker/api/types/events/events.go

```diff
@@ -19,6 +19,8 @@ const (
 	NodeEventType = "node"
 	// SecretEventType is the event type that secrets generate
 	SecretEventType = "secret"
+	// ConfigEventType is the event type that configs generate
+	ConfigEventType = "config"
```

Comment:
Brought in by:
* https://github.com/moby/moby/pull/34032 make engine support cluster config event